lib/malan/accounts/user.ex

```diff
@@ -75,15 +75,17 @@ defmodule Malan.Accounts.User do
     field :ethnicity, :string, virtual: true
 
     embeds_many :tos_accept_events, TosAcceptEvent, on_replace: :raise do
-      @derive {Jason.Encoder, except: [:__meta__]}
+      #@derive {Jason.Encoder, except: [:__meta__]}
```

Comment:
This will be removed soon once verified that we don't need it.  The tests all pass but if we see an unwanted new key creeping in we will want to use this to exclude it, and it's hard to find in the documentation.